app/Http/Controllers/SitemapController.php

```diff
@@ -22,17 +22,17 @@ public function index(){
         return Sitemap::index();
     }
 
-    Public function users(){
+    Public function users() {
         $users = User::all();
 
-        foreach ($users as $user){
+        foreach ($users as $user) {
             Sitemap::addTag(route('users.show', $user->id), $user->created_at, 'monthly', '0.8');
         }
 
         return Sitemap::render();
     }
 
-    public function games(){
+    public function games() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->